lib/sync/installation.js

```diff
@@ -44,23 +45,22 @@ const upperFirst = str => str.substring(0, 1).toUpperCase() + str.substring(1);
 const getCursorKey = jobType => `last${upperFirst(jobType)}Cursor`;
 
 /**
- * @param {import('probot').Application} app - The probot application
  * @param {any} queues - TBD
  */
-const processInstallation = (app, queues) => {
-  const updateJobStatus = async (jiraClient, job, edges, task, repositoryId) => {
+const processInstallation = (queues) => {
+  const updateJobStatus = async (jiraClient, job, edges, task, repositoryId, app) => {
     const { installationId, jiraHost } = job.data;
     // Get a fresh subscription instance
     const subscription = await Subscription.getSingleInstallation(jiraHost, installationId);
 
     // handle promise rejection when an org is removed during a sync
     if (subscription == null) {
-      app.log('Organization has been deleted. Other active syncs will continue.');
+      app.log.info('Organization has been deleted. Other active syncs will continue.');
```

Comment:
I don't think app.log.info is the function in app.log. Can you just keep it app.log?